app/controllers/AddressUKController.scala

```diff
@@ -59,7 +59,7 @@ class AddressUKController @Inject() (
       "form"      -> form,
       "regime"    -> regime.toUpperCase,
       "action"    -> routes.AddressUKController.onSubmit(mode, regime).url,
-      "countries" -> countryJsonList
+      "countries" -> Json.obj("text" -> "United Kingdom", "value" -> "GB", "selected" -> true)
```

Comment:
@soljeggers Is it correct only one country

Reply:
Yeah as its the Address UK page - The users don't get the box to choose the country on this page so we default the country field to GB for them.